clang.bazelrc

```diff
@@ -1,6 +1,6 @@
-build:clang --action_env=CC=clang
-build:clang --action_env=CXX=clang++
+build:clang --action_env=CC=clang-9
+build:clang --action_env=CXX=clang++-9
 build:clang --action_env=LLVM_CONFIG=llvm-config-9
 build:clang --repo_env=LLVM_CONFIG=llvm-config-9
-build:clang --linkopt=-L/usr/lib/llvm-9/lib
```

Comment:
We need to remember to keep track of this in case the build environment goes to `clang-9.0.2` or similar. Or maybe we could get the version from `clang --version` itself?

Reply:
Ah, the comment is accidentally to the removed line instead of the added line (which was the target)...

Reply:
right. the last two lines could also be generated before the build: `llvm-config-9 --ldflags`

Reply:
fixed